huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go

```diff
@@ -133,17 +133,17 @@ resource "huaweicloud_dms_rocketmq_instance" "test" {
   broker_num        = 1
   enable_acl        = true
 }
-`, testAccDmsRocketmqInstance_Base(rName), updateName)
+`, testAccDmsRocketmqInstance_Base(name), name)
 }
 
-func testDmsRocketMQInstance_update(rName, name string) string {
+func testDmsRocketMQInstance_update(name string) string {
```

Comment:
the config of `testDmsRocketMQInstance_update` and `testDmsRocketMQInstance_basic` is same. 
remove one?

Reply:
the param of enable_acl is different